### DISCUSSION TITLE
Differents questions about OGCS

### DISCUSSION THREAD

**Discussion by ZeSpecialK** [phw198/OutlookGoogleCalendarSync#2008] [General]
Hi all,

I have several questions about OGCS :
- About the sync time/sync delay
What is the refresh frequency for events created on a calendar ? I saw that I could set sync options on software settings but the minimum schedule amount is 15 minutes. Is it true ?
- I read my calendar on both phone and computer. I create events on both to. Does I necessary install OutlookGoogleCalendarSync on both devices ?
- Does the software run on background if it closed or does I lauch the software on each session opening and maintain it open to sync calendar events ?
- If I want to donate to support the project, could I have an invoice for my accounting department ?

Thank you by advance for all your answers.

Best regards

**Comment by phw198** [comment]
Hi @ZeSpecialK 

Time Delay: Yes, correct - 15 minute schedule or a Push Sync from Outlook within 2 minutes of a change (but then minimum of 2 hourly sync with Push Sync enabled). If you know how to set up your own API keys, then the limitation is removed entirely.

Devices: OGCS is only installable on Windows. If the PC is running with Outlook and OGCS, then changes on your phone will sync.

Software: If it's closed, then no it can't run. There are options to minimise to the system tray and also change the close button to minimise instead of close.

Donate: Donation and invoicing is a bit of an odd mix, but I believe PayPal offers the ability to produce an invoice from payments received so shouldn't be a problem 🙂 

Hope that helps!
